src/beacon-api/app/utils/field_mappings.py

```
@@ -1,38 +1,116 @@
+        val = val.strip()
+        if not val:
+            return {}
+        try:
+            parsed = json.loads(val)
+            if isinstance(parsed, dict):
+                return parsed
+            if isinstance(parsed, str):
+                parsed2 = json.loads(parsed)
+                if isinstance(parsed2, dict):
+                    return parsed2
+        except Exception:
```

Ruff (0.16.3)

[warning] Do not catch blind exception: `Exception`

(BLE001)